Config/Grobularis/options.h

```diff
@@ -0,0 +1,893 @@
+#define HEIGHT_MARGIN                       10
+
+// Use ALT_HOLD_THROTTLE_MAX when below HEIGHT_MARGIN of the target height.
+// Interpolate between ALT_HOLD_THROTTLE_MAX and ALT_HOLD_THROTTLE_MIN
+// when within HEIGHT_MARGIN of the target height.
+// Use ALT_HOLD_THROTTLE_MIN when above HEIGHT_MARGIN of the target height.
+// Throttle values are from 0.0 - 1.0.
+#define ALT_HOLD_THROTTLE_MIN               0.0
+#define ALT_HOLD_THROTTLE_MAX               1.0
+
+// Use ALT_HOLD_PITCH_MAX when below HEIGHT_MARGIN of the target height.
+// Interpolate between ALT_HOLD_PITCH_MAX and ALT_HOLD_PITCH_MIN when
+// within HEIGHT_MARGIN of the target height.
+// Use ALT_HOLD_PITCH_HIGH when above HEIGHT_MARGIN of the target height.
+// Pitch values are in degrees.  Negative values pitch the plane down.
+#define ALT_HOLD_PITCH_MIN                 -15.0
+#define ALT_HOLD_PITCH_MAX                  -4.0
+#define ALT_HOLD_PITCH_HIGH                -15.0
+
+
+////////////////////////////////////////////////////////////////////////////////
+// Return To Launch Pitch Down in degrees, a real number.
+// this is the real angle in degrees that the nose of the plane will pitch downward during a return to launch.
+// it is used to increase speed (and wind penetration) during a return to launch.
+// set it to zero if you do not want to use this feature.
+// This only takes effect when entering RTL mode, which only happens when the plane loses the transmitter signal.
+#define RTL_PITCH_DOWN                      0.0
+
+
+////////////////////////////////////////////////////////////////////////////////
+// Hardware In the Loop Simulation
+// Only set this to 1 for testing in the simulator.  Do not try to fly with this set to 1!
+// See the MatrixPilot wiki for more info on using HILSIM.
+// HILSIM_BAUD is the serial speed for communications with the X-Plane plugin.  Default is
+// now 38400.  Make sure the X-Plane plugin's Setup file has its speed set to match.
+#ifndef HILSIM
+#define HILSIM                              1
+#endif
+#define HILSIM_USB                          0           // AUAV3 only (under development)
+#define HILSIM_BAUD                         38400
+
+
+////////////////////////////////////////////////////////////////////////////////
+// Flight Plan handling
+//
+// You can define your flightplan either using the UDB Waypoints format, or using UDB Logo
+// Set this to either FP_WAYPOINTS or FP_LOGO
+// The Waypoint definitions and options are located in the flightplan-waypoints.h file.
+// The Logo flight plan definitions and options are located in the flightplan-logo.h file.
+#ifndef FLIGHT_PLAN_TYPE
+#define FLIGHT_PLAN_TYPE                    FP_LOGO
+#endif
+
+////////////////////////////////////////////////////////////////////////////////
+// Waypoint handling
+
+// Move on to the next waypoint when getting within this distance of the current goal (in meters)
+#define WAYPOINT_PROXIMITY_RADIUS	40
+
```

Comment:
I've played with this several time over the years. In LOGO i want to take decisions once every second. Some calculations take more time. This setting allows to survive the delay, where i would normally overshoot the goal and a quick circle would follow. It will not matter to the flight in Hilsim (no change needed here). BTW, the default square in LOGO is really small, it will not look pretty.